### PR TITLE
reservation controller tests done

### PR DESCRIPTION
**Pull request by sevak-yer**

### CODE REVIEW

File: GG_clone_backend/Gemfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw] # Use sqlite3 as the database for Active Record
   gem 'sqlite3', '~> 1.4'
+  gem 'rspec-rails', '~> 5.0.0'
```
Comment (ArtyomKosakyan) [inline GG_clone_backend/Gemfile:33]: You can add this gem under "development/testing"

File: GG_clone_backend/spec/controllers/api/v1/cordinates_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        post :update_driver_coordinates, params: {id: 1, coordinates: {latitude: 123, longitude: 456}}
+        driver.reload
+        expect(driver.latitude).to eq(123)
+        expect(driver.longitude).to eq(456)
+    end
+  end
+
+  describe "#create_reservation" do
+    let(:driver) {
+      Driver.create!(id: 1, first_name: "test_first_name", last_name: "test_last_name", password: "test_password", car_manufacturer: "test_car_manufacturer", car_model: "test_car_model", phone_number: 123456789)
```
Comment (ArtyomKosakyan) [inline GG_clone_backend/spec/controllers/api/v1/cordinates_controller_spec.rb]: avoid lines like this (very big)

File: GG_clone_backend/Gemfile (hunk 1 above)
Comment (sevak-yer) [reply GG_clone_backend/Gemfile:33]: it is under development test

File: GG_clone_backend/spec/controllers/api/v1/cordinates_controller_spec.rb (hunk 2 above)
Comment (sevak-yer) [reply GG_clone_backend/spec/controllers/api/v1/cordinates_controller_spec.rb]: done

**Review by ArtyomKosakyan — APPROVED** [review]

File: src/components/Driver/DriverPage.js [hunk 3]
```diff
@@ -32,7 +32,8 @@ const DriverPage = () => {
     axios.post(`${baseUrl}/coordinates/confirm`, {
       id: userId,
     })
```
Comment (iwazowsky) [inline src/components/Driver/DriverPage.js:34]: not a part of testing, consider separating PR's in the future. Just a suggestion tho, everything else seems okay :)

**Review by iwazowsky — APPROVED** [review]

**Pull request merged**